<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GBCataLogDlg</class>
 <widget class="QWidget" name="GBCataLogDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>142</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GBCataLogDlg</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="3">
      <widget class="QLineEdit" name="le_model"/>
     </item>
     <item row="2" column="4">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Secrecy:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QLineEdit" name="le_Civilcode"/>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Registerway:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="le_deviceID"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="le_deviceName"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>DeviceID:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLineEdit" name="le_omner"/>
     </item>
     <item row="0" column="6">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>PlatformAddr:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Civilcode:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="le_manufacturer"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="le_Address"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Manufacturer:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QLineEdit" name="le_Secrecy"/>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Address:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QLineEdit" name="le_Registerway"/>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Owner:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Model:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <widget class="QLineEdit" name="le_PlatformAddr"/>
     </item>
     <item row="1" column="6">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>PlatformPort:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="7">
      <widget class="QLineEdit" name="le_PlatformPort"/>
     </item>
     <item row="2" column="6">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Status:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="7">
      <widget class="QLineEdit" name="le_Status"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pb_search">
       <property name="text">
        <string>设备目录查询</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
